app/routers/share.py

```diff
@@ -62,6 +62,12 @@ def send_in_app_invitation(
     return True
 
 
+def decline(invitation: Invitation, session: Session) -> None:
+    """decline an invitation."""
+    invitation.status = 'declined'
```

Comment:
What are the options for invitation status? (consider Enum :))

Reply:
Great tip, thanks!